packages/plugins/users-permissions/tests/content-api/auth.test.e2e.js

```diff
@@ -45,7 +45,9 @@ describe('Auth API', () => {
         body: {},
       });
 
-      expect(res.statusCode).toBe(403);
+      expect(res.statusCode).toBe(400);
```

Comment:
This should be a 401 since no auth was provided, or a 403 like it was originally, even arguably a 422, but 400 seems wrong.